### ISSUE TITLE
investigate: weekly test of contact form via gh action

### ISSUE THREAD

**Issue by edm00se** [ruthquilts/ruthquilts.github.io#46]
can schedule a GH action for crontab style schedule

https://help.github.com/en/actions/automating-your-workflow-with-github-actions/events-that-trigger-workflows#scheduled-events-schedule

action should:
- run weekly
- perform test contact form submission
- ensure PASS/FAIL based on network response
- check if cached response? (aka if sw gets overly aggressive again)

**Comment by edm00se** [comment]
A crontab value of `3 3 * * 0` would do the task weekly on Sunday morning at 3:03am.

ref: https://corntab.com/?c=3_3_*_*_0

**Comment by edm00se** [comment]
Can write test w/ [Cypress GH action for a docker container](https://github.com/cypress-io/github-action#docker-image). Just need to specify cron schedule and spec>

Action config should probably look like this:

```yml
name: Cypress tests
on:
  schedule:
    - cron:  '3 3 * * 0'
jobs:
  cypress-run:
    name: Cypress run
    runs-on: ubuntu-16.04
    # Cypress Docker image with Chrome v78 and Firefox v70 pre-installed
    container: cypress/browsers:node12.13.0-chrome78-ff70
    steps:
      - name: Checkout
        uses: actions/checkout@v1

      - name: Cypress run
        uses: cypress-io/github-action@v1
        with:
          spec: cypress/contact.spec.js
```

**Comment by edm00se** [comment]
Fixed by cbca81c;
